src/utils/constants.js

```diff
@@ -0,0 +1,138 @@
+export const ALERT_TYPES = [
+    'warning',
+    'error',
+    'success',
+    'information'
+];
+
+export const BADGE_TYPES = [
+    'success',
+    'warning',
+    'error'
+];
+export const LABEL_TYPES = [
+    'success',
+    'warning',
+    'error'
+];
+
+export const STATUS_TYPES = [
+    'success',
+    'warning',
+    'error',
+    'available',
+    'away',
+    'busy',
+    'offline'
+];
+
+export const BUTTON_OPTIONS = [
+    'emphasized',
+    'light',
+    'shell'
+];
+export const BUTTON_TYPES = [
+    'standard',
+    'positive',
+    'negative',
+    'medium'
+];
+
+export const FORM_MESSAGE_TYPES = [
+    'error',
+    'warning',
+    'help'
+];
+
+
+export const ICON_SIZES = [
+    's',
+    'm',
+    'l',
+    'xl'
+];
+
+export const IDENTIFIER_COLORS = [
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9
+];
+
+export const IDENTIFIER_MODIFIERS = [
+    '',
+    'circle',
+    'transparent'
+];
+
+export const IDENTIFIER_SIZES = [
+    'xxs',
+    'xs',
+    's',
+    'm',
+    'l',
+    'xl',
+    'xxl'
+];
+
+export const IMAGE_SIZES = [
+    's',
+    'm',
+    'l'
+];
+
+export const IMAGE_TYPES = [
+    'circle'
+];
+
+export const INLINE_HELP_PLACEMENTS = [
+    'bottom-right',
+    'bottom-left',
+    'right',
+    'left',
+    'bottom-center'
+];
+
+export const INPUT_GROUP_ADDON_POSITIONS = [
+    'before',
+    'after'
+];
+
+export const INPUT_GROUP_TYPES = [
+    'text',
+    'number',
+    'search'
+];
+
+export const GRID_COLUMNS = [
+    1,
+    2,
+    3,
+    4,
+    5,
+    6
+];
+
+export const GRID_ROWS = [
+    1,
+    2,
+    3,
+    4,
+    5,
+    6
+];
+
+export const POPOVER_ALIGNMENTS = [
+    'right'
+];
+
+export const TOGGLE_SIZES = [
+    'xs',
+    's',
+    'l'
+];
```

Comment:
Am I the only one that finds it interesting there's no `m` in this list?

Reply:
Yes. The docs say the sizes are `--xs` `--s` normal and `--l`. https://sap.github.io/fundamental/components/toggle.html. We could just make `--m` map to the default class. What do you think?